ocaml/xapi/cert_distrib.ml

```diff
@@ -457,7 +457,10 @@ let ( (get_local_ca_certs : unit -> WireProtocol.certificate_file list)
      * executed on [path] *)
     Sys.readdir path
     |> Array.to_list
-    |> List.filter (Fun.flip Filename.check_suffix "pem")
+    |> List.filter (fun x ->
```

Comment:
Minor detail: a less generic name than `x` would be better - something like `file`, `fname`, `name`, or similar. Below `filename` is used.